src/dask_awkward/core.py

```diff
@@ -739,8 +745,28 @@ def __getitem__(self, where: Any) -> Any:
 
         return self._getitem_single(where)
 
+    def _call_behavior(self, method_name: str, *args: Any, **kwargs: Any) -> Any:
+        try:
+            getattr(self._meta, method_name)
+        except AttributeError:
+            raise ValueError(f"{method_name} is not available to this collection.")
```

Comment:
Suggest that this ca still be an AttributeError

Reply:
yea makes sense